constraint/pkg/client/drivers/local/compilers.go

```diff
@@ -0,0 +1,242 @@
+package local
+
+import (
+	"fmt"
+	"sync"
+
+	clienterrors "github.com/open-policy-agent/frameworks/constraint/pkg/client/errors"
+	"github.com/open-policy-agent/frameworks/constraint/pkg/core/templates"
+	"github.com/open-policy-agent/frameworks/constraint/pkg/regorewriter"
+	"github.com/open-policy-agent/opa/ast"
+)
+
+// Compilers is a threadsafe store of Compilers for ConstraintTemplates.
+type Compilers struct {
+	mtx sync.RWMutex
+
+	// compilers is a map from target name to a map from kind to compiler.
```

Comment:
nit: kind -> constraint template kind

Reply:
Changed to "Constraint kind" as all ConstraintTemplates have the same "kind".